src/devices/Kt0803/Kt0803.cs

```diff
@@ -0,0 +1,378 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Kt0803
```

Comment:
Please make the namespace categorize this device (i.e. Iot.Device.Radio or something)